### PR TITLE
Add CORS headers for MapFishPrint v3

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: c2cgeoportal/views/printproxy.py [hunk 1]
```diff
@@ -183,7 +185,7 @@ def report_create(self):
             "%s/report.%s" % (
                 self.config['print_url'],
                 self.request.matchdict.get('format')
-            )
+            ),
```
Comment (asaunier) [inline c2cgeoportal/views/printproxy.py]: What is the purpose of these extra commas?

File: c2cgeoportal/views/proxy.py [hunk 2]
```diff
@@ -158,18 +158,30 @@ def _proxy(self, url, params=None, method=None, cache=False, body=None, headers=
...
         cache = kwargs.get("cache", False)
         if cache is True:
-            resp, content = self._proxy_cache(*args, **kwargs)
+            resp, content = self._proxy_cache(url, **kwargs)
         else:
-            resp, content = self._proxy(*args, **kwargs)
+            resp, content = self._proxy(url, **kwargs)
 
-        return self._build_responce(resp, content, cache, service_name)
+        return self._build_responce(
```
Comment (asaunier) [inline c2cgeoportal/views/proxy.py]: responce => response

File: c2cgeoportal/views/printproxy.py (hunk 1 above)
Comment (sbrunner) [inline c2cgeoportal/views/printproxy.py]: Just to be ready to add extra arguments :-)

File: c2cgeoportal/views/proxy.py (hunk 2 above)
Comment (sbrunner) [inline c2cgeoportal/views/proxy.py]: Fixed :-)

**Comment by asaunier** [comment]
OK for me.

**Pull request merged**